src/main/java/com/jwplayer/jwplatform/JWPlatformClient.java

```diff
@@ -188,26 +207,34 @@ private JSONObject uploadVideo(final String uploadPath, final String localFilePa
   }
 
   /**
-   * see {@link #request(String, Map, boolean, String)}.
+   * see {@link #request(String, Map, boolean, String, Map)}.
    */
   public JSONObject request(final String path) throws JWPlatformException {
     return this.request(path, new HashMap<>());
   }
 
   /**
-   * see {@link #request(String, Map, boolean, String)}.
+   * see {@link #request(String, Map, boolean, String, Map)}.
    */
   public JSONObject request(final String path, final Map<String, String> params)
           throws JWPlatformException {
-    return this.request(path, params, false,"Get");
+    return this.request(path, params, false, "Get", new HashMap<>());
   }
 
   /**
-   * see {@link #request(String, Map, boolean, String)}.
+   * see {@link #request(String, Map, boolean, String, Map)}.
    */
   public JSONObject request(final String path, final String requestType)
-          throws JWPlatformException {
-    return this.request(path, new HashMap<>(), false, requestType);
+      throws JWPlatformException {
+    return this.request(path, new HashMap<>(), false, requestType, new HashMap<>());
+  }
+
+  /**
+   * see {@link #request(String, Map, boolean, String, Map)}.
+   */
+  public JSONObject request(final String path, final Map<String, String> params, final boolean isBodyParams, final String requestType)
```

Comment:
Don't update this one. It'd be a breaking change. If someone was using this request() function, it'd now be gone. Overload one more with the added header param.